src/profile/components/profileInformation/profileInformationAccountManagementLinkUtils.ts

```diff
@@ -52,3 +56,7 @@ export function getAmrUrl(authenticationMethodReference: AMRStatic): string {
       );
   }
 }
+
+export function hasTunnistusSuomiFiAmr(profile: Profile | null): boolean {
+  return extractSingleAmr(profile) === tunnistusSuomifiAMR;
```

Comment:
This should check if `tunnistusSuomifiAMR` is present _anywhere_ in the `profile`'s `amr` array.

Reply:
When does the amr array has length !== 1? The spec says it is possible, but in our environments?

From the amr spec: "Parties using this claim will need to agree upon the meanings of the values used, which may be context-specific." Is there an agreement somewhere?

UI does not process more than one amr value. For example the ProfileInformation component renders only one ProfileInformationAccountManagementLink. If there are no amr values, the component is not rendered so user won't notice it.

Reply:
Currently I guess there's only one `amr` value. I'm not aware of any case where there would be more than one value in the list. I just thought that finding the correct value from anywhere in the array would require about the same amount of code, so why not do it.

Reply:
Number of items in the "amr" array affects other places too. Now it can be assumed there is 0 or 1 item and the value can just be picked. Otherwise all checks should search if the value exists anywhere in the array.

Reply:
Got it. Since that's the case, I guess the multi-value support can be left for another time.